### PR TITLE
Issue 140 (Get on Fabric collection is failing )

### PR DESCRIPTION
**Pull request by a-ajith**

### CODE REVIEW

**Review by hemantha-beecherla — APPROVED** [review]
LGTM

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

File: svc-fabrics/fabrics/addfabric.go [hunk 1]
```diff
@@ -48,11 +50,29 @@ func AddFabric(req *fabricsproto.AddFabricRequest) response.RPC {
...
+				[]interface{}{"IP Address", plugin.IP}, nil)
+		}
+		deviceIPAddress := fmt.Sprintf("%v", addr[0])
+
+		if deviceIPAddress == address {
 			pluginID = plugin.ID
 			break
 		}
 	}
+	if pluginID == "" {
```
Comment (irappa-pattar) [inline svc-fabrics/fabrics/addfabric.go:72]: please log some message then return

**Review by irappa-pattar — APPROVED** [review]

File: svc-fabrics/fabrics/addfabric.go (hunk 1 above)
Comment (a-ajith) [reply svc-fabrics/fabrics/addfabric.go:72]: Done

**Pull request merged**